.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can you remove these files? They're specific to your environment and should not be included here.